prov/efa/src/rdm/efa_rdm_pke_utils.c

```diff
@@ -61,34 +62,36 @@ ssize_t efa_rdm_pke_init_payload_from_ope(struct efa_rdm_pke *pke,
 	assert(tx_iov_index < ope->iov_count);
 	assert(tx_iov_offset < ope->iov[tx_iov_index].iov_len);
 	iov_mr = ope->desc[tx_iov_index];
-
+	p2p_available = false;
+	inline_send = false;
 	/* When using EFA device, EFA device can access memory that
```

Comment:
Since we are removing `efa_can_access_memory`, I think we can remove these comments, just explain how copy can be removed based on the p2p_available and inline_send status later.